kolibri/core/fields.py

```diff
@@ -25,7 +25,8 @@ def create_timezonestamp(value):
         tz = value.tzinfo.zone
     else:
         tz = timezone.get_current_timezone().zone
-    date_time_string = value.strftime(date_time_format)
+        value = timezone.make_aware(value, timezone.get_current_timezone())
+    date_time_string = value.astimezone(pytz.utc).strftime(date_time_format)
```

Comment:
Yes - the other change was due to the fact that Travis was running its tests with a naive datetime for some reason (although it's possible that it doesn't always do that) - can add the suggested test!